ydb/library/yql/sql/v1/sql_query.cpp

```diff
@@ -2660,6 +2660,75 @@ TNodePtr TSqlQuery::Build(const TSQLv1ParserAST& ast) {
     WarnUnusedNodes();
     return result;
 }
+
+TNodePtr TSqlQuery::Build(const std::vector<::NSQLv1Generated::TRule_sql_stmt_core>& statements) {
```

Comment:
Are we sure that it's better to split query into statements here in parser? Did you check what we have in several statements query after pasing now? Maybe it is more convenient to split it into statements when we have TNodePtr pointing to root of the query?

Reply:
test